parity/url.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Cross-platform open url in default browser
+
+#[cfg(windows)]
+mod shell {
+	extern crate winapi;
+
+	use self::winapi::*;
+	extern "system" {
+		pub fn ShellExecuteA(
+			hwnd: HWND, lpOperation: LPCSTR, lpFile: LPCSTR, lpParameters: LPCSTR, lpDirectory: LPCSTR,
+			nShowCmd: c_int
+		) -> HINSTANCE;
+	}
+
+	pub use self::winapi::SW_SHOWNORMAL as Normal;
+}
+
+#[cfg(windows)]
```

Comment:
just a question,

is `#[cfg(windows)]` equal to  `#[cfg(target_os="windows")]` ?

therefore are there similar options for linux/macos `#[cfg(linux)]` / `#[cfg(macos)]`?